src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
Well, now that I think of this again, I do not think we should evaluate the flag for this. Such an error imo should never be silenced.